src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -262,13 +262,6 @@ public void DrawEllipse(IBrush brush, IPen pen, Rect rect)
         public void DrawGlyphRun(IBrush foreground, IGlyphRunImpl glyphRun)
         {
             if (glyphRun.FontRenderingEmSize.IsNearlyEqual(0)) return;
-            if (!glyphRun.FontRenderingEmSize.IsNearlyEqual(1))
```

Comment:
Seems now NotSupportedRequestCode.DrawGlyphRunWithNonDefaultFontRenderingEmSize can be deleted